example/docker-compose.prod.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  webapp:
+    environment:
+      - FLASK_ENV=production
+
+  nginx:
+    volumes:
+      - ./backend-nginx-prod.yml:/etc/appconf/nginx.yml:ro
+      - ./static:/var/www/webapp-static:ro
+    environment:
+      - LETSENCRYPT_EMAIL=Marek.Czaplicki@dlabs.ai
```

Comment:
I'd change that to something closer to example@example.com. Otherwise, you might find yourself spammed ;)